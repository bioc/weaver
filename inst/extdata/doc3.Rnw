%
% NOTE -- ONLY EDIT THE .Rnw FILE!!!  The .tex file is
% likely to be overwritten.
%
\documentclass[11pt]{article}

\usepackage{times}
\usepackage{hyperref}

\usepackage[authoryear,round]{natbib}
\usepackage{times}
\usepackage{comment}

\textwidth=6.2in
\textheight=8.5in
%\parskip=.3cm
\oddsidemargin=.1in
\evensidemargin=.1in
\headheight=-.3in


\newlength{\smallfigwidth}
\setlength{\smallfigwidth}{6cm}

\newcommand{\scscst}{\scriptscriptstyle}
\newcommand{\scst}{\scriptstyle}

\newcommand{\Rfunction}[1]{{\texttt{#1}}}
\newcommand{\Robject}[1]{{\texttt{#1}}}
\newcommand{\Rpackage}[1]{{\textsf{#1}}}
\newcommand{\Rclass}[1]{{\textit{#1}}}

\newcommand{\MAT}[1]{{\bf #1}}
\newcommand{\VEC}[1]{{\bf #1}}

\newcommand{\Amat}{{\MAT{A}}}

%%notationally this is going to break
\newcommand{\Emat}{{\MAT{E}}}
\newcommand{\Xmat}{{\MAT{X}}}
\newcommand{\Xvec}{{\VEC{X}}}
\newcommand{\xvec}{{\VEC{x}}}


\newcommand{\Zvec}{{\VEC{Z}}}
\newcommand{\zvec}{{\VEC{z}}}

\newcommand{\calG}{\mbox{${\cal G}$}}

\bibliographystyle{plainnat}

\title{Collection of Edge Cases for expression dependency}
\author{S. Falcon}

\begin{document}

\maketitle

\section{Introduction}

This is doc3.  Try to excercise the dependency tracking stuff.

<<t1, cache=TRUE>>=
bDat <- 1:10
x <- c(1, 2)
bDat <- bDat[x]

@ 

<<t2, cache=TRUE>>=
cDat <- bDat * 2
@ 

<<s1, cache=TRUE>>=
a <- 2
b <- a + 2
@ 

<<s2, cache=TRUE>>=
b2 <- paste(1:2, b)
@ 

<<s3>>=
a
b2
@ 

\end{document}
