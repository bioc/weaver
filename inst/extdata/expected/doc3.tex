%
% NOTE -- ONLY EDIT THE .Rnw FILE!!!  The .tex file is
% likely to be overwritten.
%
\documentclass[11pt]{article}

\usepackage{times}
\usepackage{hyperref}

\usepackage[authoryear,round]{natbib}
\usepackage{times}
\usepackage{comment}

\textwidth=6.2in
\textheight=8.5in
%\parskip=.3cm
\oddsidemargin=.1in
\evensidemargin=.1in
\headheight=-.3in


\newlength{\smallfigwidth}
\setlength{\smallfigwidth}{6cm}

\newcommand{\scscst}{\scriptscriptstyle}
\newcommand{\scst}{\scriptstyle}

\newcommand{\Rfunction}[1]{{\texttt{#1}}}
\newcommand{\Robject}[1]{{\texttt{#1}}}
\newcommand{\Rpackage}[1]{{\textsf{#1}}}
\newcommand{\Rclass}[1]{{\textit{#1}}}

\newcommand{\MAT}[1]{{\bf #1}}
\newcommand{\VEC}[1]{{\bf #1}}

\newcommand{\Amat}{{\MAT{A}}}

%%notationally this is going to break
\newcommand{\Emat}{{\MAT{E}}}
\newcommand{\Xmat}{{\MAT{X}}}
\newcommand{\Xvec}{{\VEC{X}}}
\newcommand{\xvec}{{\VEC{x}}}


\newcommand{\Zvec}{{\VEC{Z}}}
\newcommand{\zvec}{{\VEC{z}}}

\newcommand{\calG}{\mbox{${\cal G}$}}

\bibliographystyle{plainnat}

\title{Collection of Edge Cases for expression dependency}
\author{S. Falcon}

\usepackage{Sweave}
\begin{document}

\maketitle

\section{Introduction}

This is doc3.  Try to excercise the dependency tracking stuff.

\begin{Schunk}
\begin{Sinput}
> bDat <- 1:10
> x <- c(1, 2)
> bDat <- bDat[x]
\end{Sinput}
\end{Schunk}

\begin{Schunk}
\begin{Sinput}
> cDat <- bDat * 2
\end{Sinput}
\end{Schunk}

\begin{Schunk}
\begin{Sinput}
> a <- 2
> b <- a + 2
\end{Sinput}
\end{Schunk}

\begin{Schunk}
\begin{Sinput}
> b2 <- paste(1:2, b)
\end{Sinput}
\end{Schunk}

\begin{Schunk}
\begin{Sinput}
> a
\end{Sinput}
\begin{Soutput}
[1] 2
\end{Soutput}
\begin{Sinput}
> b2
\end{Sinput}
\begin{Soutput}
[1] "1 4" "2 4"
\end{Soutput}
\end{Schunk}

\end{document}
